Foundation/ProcessInfo.swift

```diff
@@ -104,12 +104,24 @@ open class ProcessInfo: NSObject {
         }
         versionString = productVersion._swiftObject
 #elseif os(Windows)
-        var siVersionInfo: OSVERSIONINFOW = OSVERSIONINFOW()
-        siVersionInfo.dwOSVersionInfoSize = DWORD(MemoryLayout<OSVERSIONINFOEXW>.size)
-        if GetVersionExW(&siVersionInfo) == FALSE {
-          return OperatingSystemVersion(majorVersion: fallbackMajor, minorVersion: fallbackMinor, patchVersion: fallbackPatch)
+        guard let ntdll = "ntdll.dll".withCString(encodedAs: UTF16.self, { LoadLibraryExW($0, nil, DWORD(LOAD_LIBRARY_SEARCH_SYSTEM32)) }) else {
+            return OperatingSystemVersion(majorVersion: fallbackMajor, minorVersion: fallbackMinor, patchVersion: fallbackPatch)
+        }
+        defer { FreeLibrary(ntdll) }
+        typealias RtlGetVersion = @convention(c) (UnsafeMutablePointer<RTL_OSVERSIONINFOW>) -> NTSTATUS
+        guard let rtlGetVersion = unsafeBitCast(GetProcAddress(ntdll, "RtlGetVersion"), to: Optional<RtlGetVersion>.self) else {
```

Comment:
Can you name this `pfnRTLGetVersion` instead please?  It makes it easier to find the references when the names match the actual function name.